includes/jira.codehaus.org.inc

```diff
@@ -7,8 +7,9 @@
 ##################################
 # Owned by: Olivier Gaudin       #
 ##################################
-RewriteRule "^/browse/SONAR(.*)$" "http://jira.sonarsource.com/browse/SONAR$1" [L]
-RewriteRule "^/browse/MSONAR(.*)$" "http://jira.sonarsource.com/browse/MSONAR$1" [L]
+#RewriteRule "^/browse/SONARSCTFS-(.*)$" "http://jira.sonarsource.com/browse/SONARTFVC-$1" [R=301,L]
```

Comment:
@hgomez-sonarsource Here are the mappings, there are just 4 of them in the end:
* SONARSCTFS-1 -> SONARTFVC-12
* SONARSCTFS-2 -> SONARTFVC-14
* SONARSCTFS-3 -> SONARTFVC-13
* SONARSCTFS-4 -> SONARTFVC-11